src/Giraffe/Csrf.fs

```diff
@@ -0,0 +1,155 @@
+namespace Giraffe
+
+/// <summary>
+/// CSRF (Cross-Site Request Forgery) protection helpers for Giraffe.
+/// Provides anti-forgery token generation and validation.
+/// </summary>
+[<RequireQualifiedAccess>]
+module Csrf =
```

Comment:
Something else we can add to the documentation later is to mention the ASP.NET service: https://learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-9.0#antiforgery-with-minimal-apis.